### ISSUE TITLE
bug?

### ISSUE THREAD

**Issue by barzanisar** [LiljaAdam/gasp#6]
/home/barza/gasp/gasp/utils/denoising_dino_feature_extractor.py 
line 379 (Forward function)
image = torch.cat(image, dim=0) 
This returns error. One image in the list is of a different size [1, 3, 770, 770] and the rest are [1,3,770,1022]

**Comment by barzanisar** [comment]
Hey,

Can you please address this bug? Thank you!

**Comment by LiljaAdam** [comment]
Hi, 

Sorry for the delayed response, I am travelling.

I believe this is due to the vertical forward looking camera in the AV2 dataset, right?

I'll try to fix this in the next update. In the meanwhile, it should work by
1) using batch size 1 to create the reduction matrix (using all cameras)
2) start extraction job with ONLY forward looking camera (any batch size)
3) start extraction job with the rest of the cameras (any batch size)

Sorry for the inconvenience.
